src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -73,7 +73,7 @@ public enum DisplayMode {
         PROPERTIES
     }
 
-    private final ObjectProperty<DisplayMode> displayMode = new SimpleObjectProperty<>(DisplayMode.CLIENTS);
+    private final ReadOnlyObjectWrapper<DisplayMode> displayMode = new ReadOnlyObjectWrapper<>(DisplayMode.CLIENTS);
```

Comment:
maybe dropping a short comment to explain why the initial display mode is set to clients in this part of the code will make it clearer. 

e.g. `our client grid expected behavior is to show both buyers and sellers on start up`